packages/create-sitecore-jss/src/templates/angular-xmcloud/package.json

```diff
@@ -1,10 +1,11 @@
-{
-  "config": {
-    "sitecoreDistPath": "/dist"
-  },
-  "scripts": {
-    "postbuild:server": "npm-run-all --serial prepare:build prepare:proxy-build",
-    "prepare:build": "move-cli ./dist/main.js ./dist/server.bundle.js",
-    "prepare:proxy-build": "ts-node --project src/tsconfig.webpack-server.json ./scripts/proxy-build.ts"
-  }
-}
+{
+  "config": {
+    "sitecoreDistPath": "/dist",
+    "relativeProxyPath": "<%- helper.relativeProxyAppDestination %>"
```

Comment:
Do we need a new property here?  It's unused.
We have a value in _.env_ - PROXY_BUILD_PATH

Reply:
I added this for demo puproses mainly, before PROXY_BUILD_PATH was merged into dev, let me revert it